examples/ssh/iperf3.json

```diff
@@ -41,6 +41,18 @@
             "name": "Ubuntu",
             "version": "22.04"
         }
+    ],
+    "parameters": [
+        {
+            "value_type": "LIST",
```

Comment:
This would assume all servers will have the same port? 

I did not observe this behaviour for the benchmark servers I saw